{
  "Faults": [
    {
      "latitude": 51.358215385417424,
      "type": "LV",
      "reference": "KN4863",
      "etaUpdate": true,
      "affectedCustomerCount": 15,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "09e77d1d-c6f5-40a1-8be5-fc2383fa2ffb",
      "minLongitude": -0.9237,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.9038417840342742,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-02-13T03:08:00.000+0000",
      "estimatedRestoration": "2023-02-13T10:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KN4863'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KN4863",
      "maxLongitude": -0.898,
      "minLatitude": 51.3551,
      "name": "RG27 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG27 0QY",
        "RG27 0QZ",
        "RG27 0RA",
        "RG40 4QT"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.9237,
              51.364
            ],
            [
              -0.898,
              51.364
            ],
            [
              -0.898,
              51.3551
            ],
            [
              -0.9237,
              51.3551
            ],
            [
              -0.9237,
              51.364
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-02-13T00:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.364,
      "etrUpdate": true
    },
    {
      "latitude": 51.35153975492289,
      "type": "LV",
      "reference": "KN4892",
      "etaUpdate": true,
      "affectedCustomerCount": 7,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "6a79e027-28d1-4192-afca-fca17d0937b1",
      "minLongitude": -0.9003,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8919959911338324,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-02-13T03:04:00.000+0000",
      "estimatedRestoration": "2023-02-13T10:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KN4892'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KN4892",
      "maxLongitude": -0.8869,
      "minLatitude": 51.3465,
      "name": "RG27 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG27 0PL",
        "RG27 0QH",
        "RG27 0QJ",
        "RG27 0QL",
        "RG27 0QP"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.9003,
              51.355
            ],
            [
              -0.8869,
              51.355
            ],
            [
              -0.8869,
              51.3465
            ],
            [
              -0.9003,
              51.3465
            ],
            [
              -0.9003,
              51.355
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.355,
      "etrUpdate": true
    },
    {
      "latitude": 51.35333067923312,
      "type": "LV",
      "reference": "KN4901",
      "etaUpdate": true,
      "affectedCustomerCount": 4,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "5ef61282-6f8b-47b9-bcc2-2f0d8a747c0b",
      "minLongitude": -0.9237,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8988315710949877,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-02-12T03:05:00.000+0000",
      "estimatedRestoration": "2023-02-13T10:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KN4901'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KN4901",
      "maxLongitude": -0.8819,
      "minLatitude": 51.3408,
      "name": "RG27 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG27 0PS",
        "RG27 0PT",
        "RG27 0PW",
        "RG27 0QY",
        "RG27 0QZ",
        "RG27 0RA",
        "RG40 4QT"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.9237,
              51.364
            ],
            [
              -0.8819,
              51.364
            ],
            [
              -0.8819,
              51.3408
            ],
            [
              -0.9237,
              51.3408
            ],
            [
              -0.9237,
              51.364
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.364,
      "etrUpdate": true
    },
    {
      "latitude": 51.38252577531193,
      "type": "LV",
      "reference": "KN4906",
      "etaUpdate": true,
      "affectedCustomerCount": 11,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "7a420fb9-ca02-46d4-8bfa-96d6fab9e4d1",
      "minLongitude": -2.1437,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.141789992988429,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-02-13T06:13:00.000+0000",
      "estimatedRestoration": "2023-02-13T11:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KN4906'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KN4906",
      "maxLongitude": -2.1374,
      "minLatitude": 51.381,
      "name": "SN12 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SN12 8AE",
        "SN12 8AL",
        "SN12 8AN",
        "SN12 8AS",
        "SN12 8AT"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.1437,
              51.384
            ],
            [
              -2.1374,
              51.384
            ],
            [
              -2.1374,
              51.381
            ],
            [
              -2.1437,
              51.381
            ],
            [
              -2.1437,
              51.384
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.384,
      "etrUpdate": true
    },
    {
      "latitude": 56.74137500113253,
      "type": "HV",
      "reference": "KN4905",
      "etaUpdate": true,
      "affectedCustomerCount": 156,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "618ba4c5-6383-4f71-829d-573fee4a4292",
      "minLongitude": -2.7847,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.7159708080405327,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-02-13T06:18:00.000+0000",
      "estimatedRestoration": "2023-02-13T09:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KN4905'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KN4905",
      "maxLongitude": -2.6507,
      "minLatitude": 56.7057,
      "name": "DD9 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "DD9 6RA",
        "DD9 6RB",
        "DD9 6RD",
        "DD9 6RE",
        "DD9 6RF",
        "DD9 6RG",
        "DD9 6RH",
        "DD9 6RJ",
        "DD9 6RL",
        "DD9 6RP",
        "DD9 6RW",
        "DD9 6RX",
        "DD9 6RY",
        "DD9 6RZ",
        "DD9 6SG",
        "DD9 6SQ",
        "DD9 7RB",
        "DD9 7RD",
        "DD9 7RH",
        "DD9 7RJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.7847,
              56.7713
            ],
            [
              -2.6507,
              56.7713
            ],
            [
              -2.6507,
              56.7057
            ],
            [
              -2.7847,
              56.7057
            ],
            [
              -2.7847,
              56.7713
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-02-13T07:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 56.7713,
      "etrUpdate": true
    }
  ]
}
